src/main/java/com/datadoghq/datadog_lambda_java/DDLambda.java

```diff
@@ -1,5 +1,6 @@
 package com.datadoghq.datadog_lambda_java;
 
+import org.slf4j.MDC;
```

Comment:
Is this a standard logging solution? Or are we forcing people to add it as a dependency?

Reply:
slf4j is a standard logging facade that unifies the most common java logging frameworks (log4j, java.util.logging, logback, etc). Our users aren't forced to use it; they can get the trace ID as a raw string and incorporate it however they want.